components/InputDecimal.tsx

```diff
@@ -8,90 +8,12 @@ import { NativeSyntheticEvent, TextInput, TextInputChangeEventData, TextInputFoc
 import AdaptiveTextInput from "./AdaptiveTextInput";
 import { View } from "./Themed";
 
-export const InputDecimal = ({ callback, type, placeholder }: { callback: Function, type: string, placeholder: string }) => {
+export const InputDecimal = ({ callback, type, placeholder, initalVal }: { callback: Function, type: string, placeholder: string, initalVal?: Number }) => {
 
-    const [input, setInput] = useState('')
+    const [input, setInput] = useState(initalVal === undefined ? '' : initalVal.toFixed(2))
```

Comment:
Let's use a single component for the amount input. You can delete this file, and if you need the amount input component we are currently using to be changed in some way, let me know.

Reply:
I am going to wait till that feature is functional